### DISCUSSION TITLE
Assessing physiological factors.

### DISCUSSION THREAD

**Discussion by ranamjz** [Open-Systems-Pharmacology/Forum#972] [Questions & Problems]
Hello all,
How can i asses the physiological factors effecting renal clearance such as gfr and pgp in pksim .
Your help is much appreciated

**Comment by StephanSchaller** [comment]
@ranamjz, what do you want to access specifically?

You have access to physiological properties in your simulation via the !PArameters" Tab:
![image](https://user-images.githubusercontent.com/25477643/134639595-09191254-64af-4c3d-baa7-f0788d51fb37.png)

**Reply by ranamjz** [reply]
Thank you so much for replying,
I want to see how changing physiological parameters such as GFR and pgp expression in the  kidney,  affects the pharmacokinetic variables of my drug

**Comment by StephanSchaller** [comment]
Well, you then change the parameters and simulate and check your results.

Alternatively, you can run a Sensitivity Analysis: https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/sensitivity-analysis

**Reply by ranamjz** [reply]
Is there any possible way to change transporters and enzyme expressions aside changing the populations existing  in the library

**Reply by StephanSchaller** [reply]
Yes, certainly. Within your individual, you can set these in q quite detailed way, changing absolute reference -and relative expressions levels:
![image](https://user-images.githubusercontent.com/25477643/136763837-702b24c2-f07a-4721-be62-ee6e284c6575.png)

**Reply by ranamjz** [reply]
Thank you so much and this option is not available for a population simulation?

**Reply by StephanSchaller** [reply]
Well, the population is built on an individual, so technically, you should first set up (the expression profile of) the individual as you like, and you can then create the population. But the population has the same settings:
![image](https://user-images.githubusercontent.com/25477643/137134397-ba958109-23b9-4614-9e26-5434491dc51e.png)
